google_shopping_functions.js

```diff
@@ -1,30 +1,92 @@
-//define getItemsCount function
-//accepts full item data
-//returns the length of the items array
 var getItemsCount = function(itemData){
-	return itemData.items.length;
+	return itemData.length;
 };
+var getItems = function(objectData) {
+	return objectData.items
+}
+var data = require("./products.json")
+
+console.log("Question 1: ***************************")
+
+var getItemsByBrand = function(items, brand) {
+	var brandArray = [];
+	for (var i = 0; i < items.length; i++){
+		if(items[i].product.brand === brand){
+			brandArray.push(items[i]);
+		}
+	}
+	return brandArray
+}
+
+var itemsByBrand = getItemsByBrand(getItems(data), 'Panasonic')
+console.log(itemsByBrand);
+
+console.log("Question 2: ***************************")
+
+var getItemsByAuthor = function(items, authorName) {
+	var authorArray = [];
+	for (var i = 0; i < items.length; i++){
+		if(items[i].product.author.name.toLowerCase().indexOf(authorName) === authorName){
```

Comment:
With indexOf, we can check if a string includes a given set of characters if it does not equal -1:
```
if (items[i].product.author.name.toLowerCase().indexOf(authorName) !== -1){
```